panacea/web/controllers/pml_controller.ex

```diff
@@ -24,14 +25,22 @@ defmodule Panacea.PmlController do
     {:error, message}
   end
 
-  defp respond({:ok, drugs}, conn) do
-    conn
-    |> put_status(:ok)
-    |> json(%{drugs: drugs})
+  defp get_ddis({:ok, drugs}) do
+    {:ok, ddis} = Panacea.Asclepius.Remote.ddis(drugs)
```

Comment:
oops, I moved `ddis` out of `Panacea.Asclepius.Remote` and into `Panacea.Asclepius` while you were working on this - so the tests will fail.

Remote is now just a behaviour module.